drivers/tests/TESTS/mbed_drivers/ticker/main.cpp

```diff
@@ -206,19 +221,18 @@ void test_multi_ticker(void)
 void test_multi_call_time(void)
 {
     Ticker ticker;
-    int time_diff;
     const int attach_count = 10;
 
     for (int i = 0; i < attach_count; i++) {
         ticker_callback_flag = 0;
         gtimer.reset();
 
         gtimer.start();
-        ticker.attach_us(callback(stop_gtimer_set_flag), MULTI_TICKER_TIME_MS * 1000);
+        ticker.attach(callback(stop_gtimer_set_flag), MULTI_TICKER_TIME);
         while (!ticker_callback_flag);
-        time_diff = gtimer.read_us();
+        auto time_diff = gtimer.elapsed_time();
 
-        TEST_ASSERT_UINT32_WITHIN(TOLERANCE_US, MULTI_TICKER_TIME_MS * 1000, time_diff);
+        TEST_ASSERT_DURATION_WITHIN(TOLERANCE, MULTI_TICKER_TIME, time_diff);
```

Comment:
See previous comment

Reply:
Could you clarify this comment? As TOLERANCE here is not a function like macro as TOLERANCE_US is above , its just a constantvalue 1000us.